src/watertap_contrib/seto/multiperiod/VAGMD_batch_flowsheet_multiperiod.py

```diff
@@ -0,0 +1,385 @@
+import logging
+import pandas as pd
+import numpy as np
+
+# Pyomo imports
+from pyomo.environ import (
+    Constraint,
+    value,
+    ConcreteModel,
+    TransformationFactory,
+    units as pyunits,
+)
+from pyomo.util.calc_var_value import calculate_variable_from_constraint
+
+# IDAES imports
+from idaes.apps.grid_integration.multiperiod.multiperiod import MultiPeriodModel
+from idaes.core.util.model_statistics import degrees_of_freedom
+from idaes.core.util.exceptions import (
+    ConfigurationError,
+    UserModelError,
+    InitializationError,
+)
+import idaes.logger as idaeslog
+from idaes.core.solvers.get_solver import get_solver
+
+# WaterTAP imports
+from watertap.property_models.seawater_prop_pack import SeawaterParameterBlock
+from watertap.core.util.model_diagnostics.infeasible import *
```

Comment:
A python purist would tell you to avoid star imports (I am less strict)... but are any of these functions used? I am looking for ``print_infeasible_constraints()`` etc and I can't see them.

Reply:
This should be deleted if none are used. In fact, once debugging of the model is done, I would think this import should not be included at all.

Reply:
Thanks. It is deleted now.